skyportal/handlers/api/tag.py

```diff
@@ -51,13 +23,24 @@ def post(self):
         if not tag_text or not isinstance(tag_text, str):
             return self.error("`tag_name` must be provided as a non-empty string")
 
+        if not re.fullmatch(r"[A-Za-z0-9]+", tag_text):
```

Comment:
Can't remember what we landed on but I think this is a bit different. What I suggested what to string the string of special characters, spaces, casing, ... and such **when checking for duplication**, not completely banning them.

It's better to be strict now and loosen it up later though. So let's keep this and not change anything, and we'll think about this more thoroughly for another PR. Sounds good?